public/omni/omni-cluster-setup/registering-machines/join-machines-to-omni.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Join machines to Omni
+---
+
+Omni lets you connect existing Talos nodes—whether they’re running on bare metal, in QEMU, or on cloud instances—to your Omni environment so it can automatically take over their management.
+
+You can register a node by applying a **Machine Join Config**, which tells the node how to connect to Omni during startup.
+
+> **Note:** Once a machine joins Omni, it is managed entirely through Omni.  
+> The local Talos API is disabled for security reasons, and all future configuration changes must be made through Omni.
+
+The following section explains how to use a Machine Join Config to add your Talos nodes to Omni.
+
+## Machine Join Config
+
+The Machine Join Config is a partial configuration that lets you register existing Talos nodes with Omni.
```

Comment:
```suggestion
The Machine Join Config is a configuration that lets you register existing Talos nodes with Omni.
```

Reply:
@Unix4ever's earlier review(one of the resolved comments) added the partial adjective @rothgar